coalahtml/_coalahtml/app/controllers/logs.js

```diff
@@ -1,16 +1,12 @@
 'use strict';
 
 angular.module('coalaHtmlApp')
-  .controller('LogsCtrl', function ($scope, $http) {
+  .controller('LogsCtrl', function ($scope, $rootScope) {
     var parseCoalaProject = function(projectDir, coalaJSON) {
-      coalaJSON = coalaJSON || projectDir + $scope.Constants.coala;
+      coalaJSON = coalaJSON || projectDir + $rootScope.CONSTANTS.coala;
```

Comment:
Indeed
ᐧ

On Fri, Jun 3, 2016 at 3:23 PM, Attila <notifications@github.com> wrote:

> In coalahtml/_coalahtml/app/controllers/logs.js
> <https://github.com/coala-analyzer/coala-html/pull/52#discussion_r65683259>
> :
>
> >      var parseCoalaProject = function(projectDir, coalaJSON) {
> > -      coalaJSON = coalaJSON || projectDir + $scope.Constants.coala;
> > +      coalaJSON = coalaJSON || projectDir + $rootScope.CONSTANTS.coala;
>
> It seems you don't use coalaJSON anymore.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/coala-analyzer/coala-html/pull/52/files/b7f706d2171c77e341e20b5b680c0c87b1b38b7e#r65683259>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AHDgOdI4poyEvrLEnMfv6df5h-hmsTcKks5qH_m0gaJpZM4IslPq>
> .
>



-- 
Best regards,
Tushar Gautam
http://gtushar.co